typescript.example/examples/minimal-typecheck.ts

```diff
@@ -1,14 +1 @@
-var a;
-
-function f(x: string) {
-	return x;
-}
-
-f(a);
-
-interface X {}
-
-const b: X = a;
-const c: any = b;
-//const b: A = { foo: 5 }
-//const c: A = 512
\ No newline at end of file
+const a, b, c;
```

Comment:
Your code is working, because this change is failing the build :joy: https://travis-ci.org/MetaBorgCube/metaborg-typescript/builds/243620990#L2072